README.md

```diff
@@ -25,7 +25,17 @@ to pass these lints. The recommended lints include all the core lints.
 
 Additionally, a third lint set -
 [`package:flutter_lints`](https://pub.dev/packages/flutter_lints) - extends the
-recommended set with additional recommended Flutter-specific lints. Documentation on that can be found at [github.com/flutter/packages](https://github.com/flutter/packages/tree/main/packages/flutter_lints).
+recommended set with additional recommended Flutter-specific lints.
+Documentation on that can be found at
+[github.com/flutter/packages](https://github.com/flutter/packages/tree/main/packages/flutter_lints).
+
+The Dart team will likely not provide recommendations past `core` and
+`recommended` sets (e.g., a `strict` rule set). There are many such rule sets in
+the ecosystem however (available at [pub.dev](https://pub.dev/)).
+
+Rule set authors: consider adding the `lints` topic to your pubspec to allow
```

Comment:
Consider linking "topic" to https://dart.dev/tools/pub/pubspec#topics since developers may not be familiar with how to specify them.

Reply:
sgtm